.authors.yml

```diff
@@ -0,0 +1,33 @@
+- name: Arvid Bessen
+  email: bessen@deshaw.com
+  num_commits: 3
+  first_commit: 2020-04-03 15:38:29
+  github: ArvidJB
+- name: Melissa Weber Mendonça
+  email: melissawm@gmail.com
+  aliases:
+  - Melissa Weber Mendonca
+  - MelissaWM
+  - melissawm
```

Comment:
Shouldn't these be managed by a .mailmap file?

Reply:
No, this produces the mailmap file

Reply:
Oh cool. We should generate that file and check it into the repo.

Reply:
I think it generates it and checks it in on release

Reply:
Oh it actually commits it on release? That's interesting.